pytest.ini

```diff
@@ -1,5 +1,12 @@
 [pytest]
-addopts = --strict-config --strict-markers
+addopts =
+    --strict-config
+    --strict-markers
+    --cov blurb_it
+    --cov tests
+    --cov-report html
+    --cov-report term
+    --cov-report xml
```

Comment:
The disadvantage of putting this config in `pytest.ini` rather than `.coveragerc` is that running `pytest` locally will now run pytest under coverage -- there's now no way to "just run pytest without a coverage report".

The tests in this repo are (at the moment) extremely fast, however, so there's not really any reason to _not_ run tests under coverage. So I'm okay with this!